components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

```diff
@@ -106,9 +106,11 @@
     public static final String ASSOCIATED_ID = "associatedID";
 
     public static final String JIT_PROVISIONING_FLOW = "JITProvisioningFlow";
+    public static final String MANDATORY_CLAIM_UPDATE_FLOW = "MandatoryClaimUpdateFlow";
```

Comment:
This is only relevant for MandatoryClaimUpdat in JIT provisioning flow, right? If so we may need to change the variable name accordingly..

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3641/commits/0d4090722a9ecc9ea710c11cd1de7c8b426534b2